examples/gatsby/src/components/Products/SkuCard.js

```diff
@@ -37,7 +37,7 @@ const SkuCard = ({ sku }) => {
         {formatCurrencyString({
           value: sku.price,
           currency: sku.currency,
-          language: navigator.language,
+          language: typeof window ?? navigator.language,
```

Comment:
`typeof` returns a string so you'll always get `"object"` or `"undefined"`. We don't need this though as `formatCurrencyString` handles this. This line doesn't need to exist since we want the language to default to `navigator.language` and then default to `'en-US'` if we don't have access to it:

```suggestion
```